Makefile

```diff
@@ -42,16 +43,18 @@ down:
 debug:
 	docker-compose run --rm site bash
 
-# Enter the shell of the running dev container
+# Enter the shell of the *running* dev container
 shell:
 	docker-compose exec site bash
 
-# Build the dev container from scratch, overcoming 
-# a funky (documented) error with installed packages 
-# that may hav something to do with use of a volume
+# Build the dev container from scratch.
+# Runs packages installs a second time for both Gems and NPM to 
+# overcome inconsistent bugs with missing packages at runtime.
 setup:
 	make clean
 	docker-compose build site
+	docker-compose run --rm site bundle install
```

Comment:
For the record, I was unable to build the site without these lines. Glad to see them back.